docs/admin/etcd.md

```diff
@@ -1,49 +1,196 @@
 ---
```

Comment:
I'll add the original etcd.md to this PR @xiang90 and I can make some small changes to update that doc if you'd like